src/cascadia/TerminalControl/TermControlAP.idl

```diff
@@ -0,0 +1,14 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import "TermControl.idl";
+
+namespace Microsoft.Terminal.TerminalControl
+{
+    [default_interface]
+    runtimeclass TermControlAP :

```

Comment:
I can't stop thinking "TermControlAccessPoint" when I see this. Maybe we should rename it to `TermControlAutomationPeer`